plugins/source/pagerduty/resources/plugin/plugin.go

```diff
@@ -0,0 +1,19 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/pagerduty/client"
+	"github.com/cloudquery/plugin-sdk/plugins"
+)
+
+var (
+	Version = "development"
```

Comment:
```suggestion
	Version = "Development"
```
https://github.com/cloudquery/cloudquery/blob/main/plugins/source/aws/resources/plugin/plugin.go#L9

Reply:
Done.